packages/auth-browser/src/lib/chains/eth.ts

```diff
@@ -386,8 +391,7 @@ export const connectWeb3 = async ({
     };
 
     if (isBrowser()) {
-      // @ts-ignore
-      globalThis.litWCProvider = wcProvider;
+      litWCProvider = wcProvider;
```

Comment:
double checked that this variable is not used in other modules as it's now locally scoped instead of global